### DISCUSSION TITLE
怎么让B站直播推送可以@全员?

### DISCUSSION THREAD

**Discussion by NanaHachiKyu** [Tsuk1ko/CQ-picfinder-robot#293] [问题求助]
我尝试在加上[CQ.at(all)]
![)O8G UQP 1686J@82$DJH{U](https://user-images.githubusercontent.com/63780213/154835152-ca59204c-f5ef-4df1-9e7b-d25434e10c6c.png)
但是all没有被定义
![0FYB$BP$XI7O} 8OMNBP](https://user-images.githubusercontent.com/63780213/154835179-866d57af-0a0a-4046-9a16-41f18d341f66.png)

**Comment by Tsuk1ko** [comment]
……不是这么改的

但这个功能确实可以作为新需求，后续我增加一个设置方式吧

**Reply by NanaHachiKyu** [reply]
好耶,大佬牛逼

**Reply by Tsuk1ko** [reply]
已经在最新版支持，详见 wiki
